schemas/com.snowplowanalytics.snowplow/media_player_event_fullscreen_change/jsonschema/1-0-0

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Media player event fired immediately after the browser switches into or out of full-screen mode.",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player_event_fullscreen_change",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "label": {
```

Comment:
Given this fires for both (minimised => full screen and full screen => minimised) I think it's worth including a property that shows that direction (e.g., `fullscreen: boolean` to indicate the state the user has attempted to transition to).

Reply:
✅ added `fullscreen` property.